### ISSUE TITLE
Add Inbox notification to prompt merchant to set statement descriptor when none is set

### ISSUE THREAD

**Issue by thenbrent** [Automattic/woocommerce-payments#734]
Depends on #407.

As mentioned in the comments on pbUcTB-13-p2, if a merchant's account is using the default statement descriptor on the platform, we should implement an Inbox notification to encourage them to update the statement descriptor for their site URL.

Key benefit to present in the notification: _reduce charge backs_. Using a statement descriptor that makes it easier for customers to connect charges on their credit card statement with purchases on this store reduces the risk of a customer lodging a charge back for a valid purchase.

**Comment by kalessil** [comment]
@thenbrent : this issue seems to be obsolete - last year we implemented statement auto-generation and migrated all merchants statements. Hence closing the ticket.

**Comment by thenbrent** [comment]
There's still value prompting the merchant to set their descriptor.

For example, if a site is selling under the domain examplearbitraryreallylongdomainname.com, then I'm pretty sure the auto-generated descriptor will be `examplearbitraryreally`, which isn't ideal. The merchant may want to customize it to something more easy to recognize.

That said, we can keep this issue closed as it's sufficiently low priority for now it doesn't need to remain open. We can reopen it later if we see evidence that dispute rates are higher on accounts using the auto-generated descriptor.

**Comment by kalessil** [comment]
Thank you for sharing the use-case @thenbrent. I also like the idea to reopen this if auto-generated statements causing a higher disputes rate, great catch.
